### DISCUSSION TITLE
Is there a way to trigger a form's OnLoad() events in bulk?

### DISCUSSION THREAD

**Discussion by WesleyTheMan** [Innofactor/XrmToolBox.BulkDataUpdater#172] [Q&A]
Is there a way to trigger a form's OnLoad() events in bulk?

I believe these events are from the form's JavaScript as there are plugins triggered on first viewing a record without making any changes manually. Thank you!

**Comment by rappen** [comment]
Sorry, we can't trigger for the client-side, this is only working on the server-side.
Move it from the JS to a plugin - then you can to anything you want to! :)
